dp_check/dp_check.go

```diff
@@ -142,32 +235,103 @@ func main() {
 	if syscall.Getuid() != 0 {
 		infoLog.Println("Not running as root, some checks may fail.")
 	}
-	// Check IPv6
-	runCheck("IPv6 routes", func() error {
-		var out string
-		var err error
-		cmd := "ip -6 route show"
-		infoLog.Printf("Check IPv6 routes with:|%v|...\n", cmd)
-		if out, err = getOutput(cmd); err != nil {
+
+	// Check if dp_check is running on a VM
+	runCheck("Running on GCE VM", func() error {
```

Comment:
nit: can we rename this to `Running on GCP` ?

"Running on GCE VM" may be misleading, because this is also supposed to work e.g. on GKE